packages/data/src/redux-store/metadata/selectors.js

```diff
@@ -46,8 +57,12 @@ export function getResolutionState( state, selectorName, args ) {
  * @return {boolean | undefined} isResolving value.
  */
 export function getIsResolving( state, selectorName, args ) {
-	const resolutionState = getResolutionState( state, selectorName, args );
+	deprecated( 'wp.data.select( store ).getIsResolving', {
+		since: '6.6',
+		alternative: 'wp.data.select( store ).getResolutionState',
```

Comment:
```suggestion
		alternative: 'wp.data.select( store ).isResolving',
```

Reply:
I originally also wanted to suggest `isResolving` as the alternative, but when looking at #6600, I realized that:
- the two selectors, `isResolving` and `getIsResolving`, have been there since the beginning, added in the same commit. It's not that one of them was added only later, to create some fixed v2 of the previous one.
- the purpose of `getIsResolving` is to get the raw data from the store. The other selectors, `isResolving`, `hasResolving`, were implemented in terms of `getIsResolving` and checking its return value.

Therefore, the right alternative for `getIsResolving` is `getResolutionState`. That's the selector that returns the raw data now, and is used to implement all the others.

And by "alternative" I mean that if you were using the old selector for its real purpose, not by mistake, then the new selector to replace it with is `getResolutionState`.

Using `getIsResolving` where you really wanted to use `isResolving` was always a mistake, a misunderstanding.

Reply:
Makes sense